sktime/split/sameloc.py

```diff
@@ -19,9 +19,32 @@
 class SameLocSplitter(BaseSplitter):
     r"""Splitter that replicates loc indices from another splitter.
 
-    Takes a splitter ``cv`` and a time series ``y_template``.
```

Comment:
Nice! We should leave the "plain English" description though, and only then proceed to the math.